bin/cassandra.in.sh

```diff
@@ -41,15 +41,15 @@ cassandra_bin="$cassandra_bin:$CASSANDRA_HOME/build/classes/thrift"
 # if not set in cassandra.yaml
 cassandra_storagedir="$CASSANDRA_HOME/data"
 
-if [ -n "${JAVA_11_HOME}" ]; then
-    JAVA_HOME="${JAVA_11_HOME}"
+if [ -n "${JAVA_15_HOME}" ]; then
+    JAVA_HOME="${JAVA_15_HOME}"
 fi
 
-# verify that JAVA_HOME points to java 11
+# verify that JAVA_HOME points to java 15
 java_ver_output=`"$JAVA_HOME/bin/java" -version 2>&1 | grep '[openjdk|java] version' | awk -F'"' 'NR==1 {print $2}'`
 jvm_version=${java_ver_output%_*}
-if [ "$jvm_version" \< "11.0" ] ; then
```

Comment:
any chance we want to support java 11 + 15? is there any chance that some deployments still don't use java 15?

Reply:
especially given there are no actual code changes